### PR TITLE
Markup clean-up for pupils project (part 1)

### PR DESCRIPTION
**Pull request by pris54**
Covers 'pupil' app.
Stories 13203 and 13212 -
References to 4pin authentication removed.
HTML validates

### CODE REVIEW

File: pupil/docker-compose.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ services:
     image: mtcbeta.azurecr.io/app:latest
     environment:
       USER: node
-      AUTH_OPTION: 4pin
+      AUTH_OPTION: 5pin
```
Comment (jon-shipley) [inline pupil/docker-compose.yml]: This should be uppercase `5PIN`, if we still need it.

File: pupil/views/check/feedback.ejs [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+<header class="page-header">
     <h1 class="heading-xlarge">Give feedback</h1>
 </header>
 
 <div class="grid-row">
     <div class="column-two-thirds">
+
         <% if (Object.getOwnPropertyNames(error).length > 0) { %>
-        <div class="error-summary" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+        <div class="error-summary" role="group" tabindex="-1">
```
Comment (jon-shipley) [inline pupil/views/check/feedback.ejs]: Why remove the aria markup?

File: pupil/views/partials/_content.ejs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<main id="content" class="js-content" role="main">
+<main id="content" class="js-content">
```
Comment (jon-shipley) [inline pupil/views/partials/_content.ejs]: Why remove `role="main"`? I think the redundancy is intended looking at other gov.uk webpages.

File: pupil/views/partials/_question_content.ejs [hunk 4]
```diff
@@ -1,3 +1,3 @@
-<main id="content" class="js-content" role="main">
+<main id="content" class="js-content">
```
Comment (jon-shipley) [inline pupil/views/partials/_question_content.ejs]: Why remove `role="main"`?  Most gov.uk web pages have this.  I assume it tested better.

File: pupil/views/question-layout.ejs [hunk 5]
```diff
@@ -40,7 +40,7 @@
     </div>
 </div>
 
-<header role="banner" id="global-header" class="<% if (headerClass) { %><%= headerClass %><% } %>">
+<header id="global-header" class="<% if (headerClass) { %><%= headerClass %><% } %>">
```
Comment (jon-shipley) [inline pupil/views/question-layout.ejs]: `role="banner"` is a part of the the `govuk_template` : http://alphagov.github.io/govuk_template/example-mvp.html

File: pupil/views/warm-up/question.ejs [hunk 6]
```diff
@@ -7,7 +7,7 @@ if (typeof nextQuestion === 'number') {
 }
 %>
 <br style="clear:both"/>
-<p>
+<p class="page-header">
```
Comment (jon-shipley) [inline pupil/views/warm-up/question.ejs]: Applying a class called `page-header` to a `<p>` tag seems incongruous.

File: pupil/docker-compose.yml (hunk 1 above)
Comment (pris54) [inline pupil/docker-compose.yml]: Who can confirm if we still need it? Guy?

File: pupil/views/check/feedback.ejs (hunk 2 above)
Comment (pris54) [inline pupil/views/check/feedback.ejs]: It was the only aria entry I found. I will cover accessibility all at once after this clean-up, the aim of this PR is having consistent, valid mark-up. If this code is still valid I will restore it.

File: pupil/views/warm-up/question.ejs (hunk 6 above)
Comment (pris54) [inline pupil/views/warm-up/question.ejs]: Please suggest a better name.

File: pupil/views/partials/_question_content.ejs (hunk 4 above)
Comment (pris54) [inline pupil/views/partials/_question_content.ejs]: This roles were flagged as invalid markup. I will review roles in general as part of accessibility.

File: pupil/views/check/feedback.ejs (hunk 2 above)
Comment (jon-shipley) [inline pupil/views/check/feedback.ejs]: The code is valid, please restore it.

Comment (pris54) [inline pupil/views/check/feedback.ejs]: The one Aria insert is back Jon, would you mind approving my PR please?

**Comment by shekss** [comment]
55 scenarios (55 passed)
140 steps (140 passed)
4m2.639s

**Pull request merged**